.travis.yml

```diff
@@ -2,10 +2,12 @@ sudo: false
 
 language: go
 
+env:
+  global:
+  - GO15VENDOREXPERIMENT=1
```

Comment:
One can not vendor it. golang will try to pick it up from the GOPATH. However golang can then get 'confused' about recursive dependencies...

This question has nothing to do with godep, it is entirely a question of golang and what you put in `vendor/`.

Ingress doesn't use other packages in `contrib/` so its a non-issue here.